src/sugar3/bundle/activitybundle.py

```diff
@@ -130,13 +130,22 @@ def __init__(self, path, translated=True):
 
     def _parse_info(self, info_file):
         cp = ConfigParser()
-        if six.PY2:
-            cp.readfp(info_file)
-        else:
-            cp.read_file(info_file)
+
+        content = info_file.read()
+        if isinstance(content, bytes):
+            content = content.decode("utf-8")
+        try:
+            if content.startswith("b'") or content.startswith('b"'):
+                import ast
+                content = ast.literal_eval(content)
+        except Exception:
+            pass
+
+        cp.read_string(content, source=getattr(info_file, 'name', '<unknown>'))
```

Comment:
I'm not so sure if this is the best place to change it or to make sure the file isn't read as bytes in `get_file` to begin with.

Also, did you try directly opening the file with `cp.read_file(open(info_file))` for python3 because that's where this issue stems from it seems.

@quozl thoughts?